src/dappkit/DappKitAccountScreen.tsx

```diff
@@ -59,32 +64,44 @@ const DappKitAccountScreen = ({ route, handleContentLayout }: Props) => {
     }
   }
 
+  const handleLayout = (event: LayoutChangeEvent) => {
+    handleContentLayout(event)
+    if (event.nativeEvent.layout.height >= height) {
+      setScrollEnabled(true)
+    }
+  }
+
   return (
-    <SafeAreaView edges={['bottom']} style={styles.container} onLayout={handleContentLayout}>
-      <RequestContent
-        onAccept={handleAllow}
-        onDeny={handleCancel}
-        dappImageUrl={dappConnectInfo === DappConnectInfo.Basic ? activeDapp?.iconUrl : undefined}
-        dappName={dappKitRequest.dappName}
-        title={
-          dappConnectInfo === DappConnectInfo.Basic
-            ? t('connectToWallet', { dappName: dappKitRequest.dappName })
-            : t('confirmTransaction')
-        }
-        description={phoneNumber ? t('connectWalletInfoDappkit') : t('shareInfo')}
-        requestDetails={[
-          {
-            label: t('phoneNumber'),
-            value: phoneNumber,
-          },
-          {
-            label: t('address'),
-            value: account,
-          },
-        ]}
-        testId="DappKitSessionRequest"
-      />
-    </SafeAreaView>
+    <BottomSheetScrollView
+      style={{ maxHeight: height - BOTTOM_SHEET_DEFAULT_HANDLE_HEIGHT }}
+      scrollEnabled={scrollEnabled}
+    >
```

Comment:
nit: could we factorize this into our own BottomSheetScrollView to avoid repeating the same logic in different screens?

Reply:
ah yes, i was lazy and thought about the dappkit components as temporary 🙈 i refactored into a separate component now